src/Kaleidoscope/MacrosOnTheFly.cpp

```diff
@@ -424,10 +418,10 @@ Key MacrosOnTheFly::eventHandlerHook(Key mapped_key, byte row, byte col, uint8_t
       play_col = col;
       currentState = PICKING_SLOT_FOR_PLAY;
     }
-    return Key_NoKey;
+    return kaleidoscope::EventHandlerResult::OK;
```

Comment:
Everywhere else, `return Key_NoKey` was converted to `return kaleidoscope::EventHandlerResult::EVENT_CONSUMED`.  Was the difference here intentional?

Reply:
Nope. I'll fix it up presently.